open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/SimpleRegistry.java

```diff
@@ -33,4 +39,5 @@
      * @param pProperties the properties with which to initialize the registry.
      */
     void initialize(Toolbox pToolbox, Properties pProperties);
+
```

Comment:
Remove the extra blank line (you can get Eclipse to remove extra lines for you on save)